### ISSUE TITLE
Replaced insns should not take an opcode space

### ISSUE THREAD

**Issue by marcfedorow** [riscv/riscv-p-spec#119]
I think the followed should be fixed:
- `10. Instruction Encoding Table` contains replaced insns (e.g. rv32/smmul, rv64/add64, bitrev, etc) They should be probably removed.
- `11. Removed Instructions Due to RVB overlaps` contains only three insns. Other not fully replaced insns (i.e. bitrev, wext, etc) should be probably put there too.

Also it would be great if replaced instructions will be described **only** in the last chapter.

**Comment by Kevin-Andes** [comment]
Yes, I agree with you in both issues. Thank you very much for pointing these out. We will revise them in the future release.

**Comment by pz9115** [comment]
Out of date, mark as closed.
